frame/nomination-pools/src/lib.rs

```diff
@@ -2122,7 +2125,7 @@ pub mod pallet {
 	#[pallet::hooks]
 	impl<T: Config> Hooks<BlockNumberFor<T>> for Pallet<T> {
 		#[cfg(feature = "try-runtime")]
-		fn try_state(_n: BlockNumberFor<T>) -> Result<(), &'static str> {
+		fn try_state(_n: BlockNumberFor<T>, _: TryStateSelect) -> Result<(), &'static str> {
```

Comment:
```suggestion
		fn try_state(_n: BlockNumberFor<T>, _: frame_support::traits::TryStateSelect) -> Result<(), &'static str> {
```

Okay this should work no?